libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -283,7 +301,7 @@ public void setTitle(CharSequence title) {
             return;
         }
 
-        mWPAndroidGlueCode.setContent(title.toString(), null);
+        mWPAndroidGlueCode.setTitle(title.toString());
```

Comment:
Hmm not sure why this change is part of this PR?

Reply:
I left a note in the description of the PR :D 
```
Note:
@marecar3 the base branch corresponds to PR #9120, which also needs to be updated with latest develop to clear the commit history difference seen here.
```

Reply:
Sorry and thanks! @mzorz